uni/lib/controller/networking/network_router.dart

```diff
@@ -123,9 +124,13 @@ class NetworkRouter {
     final Map<String, String> headers = <String, String>{};
     headers['cookie'] = session.cookies;
 
+    final Duration timeoutDuration = Duration(seconds: timeout);
```

Comment:
Can this be moved to line 24 where we define the timeout? 
It makes sense the timeout already be a Duration, as well the getWithCookies getting a Duration as a parameter